test/integration/testExecute.js

```diff
@@ -137,6 +137,25 @@ describe('Execute test', function () {
       done
     );
   });
+
+  it('testDescribeOnly', function (done) {
+    const selectWithDescribeOnly = 'SELECT 1.0::NUMBER(30,2) as C1, 2::NUMBER(38,0) AS C2, \'t3\' AS C3, 4.2::DOUBLE AS C4, \'abcd\'::BINARY(8388608) AS C5, true AS C6';
+
+    function executeQueryAndVerifyResultDependOnDescribeOnly(describeOnly) {
+      connection.execute({
+        sqlText: selectWithDescribeOnly,
+        describeOnly: describeOnly,
+        complete: (err, stmt, rows) => {
+          assert.strictEqual(stmt.getColumns().length, 6);
+          //Empty rowset in response
+          assert.strictEqual(rows.length, describeOnly ? 0 : 1);
```

Comment:
let's make the `expectedReturnedRows` the parameter for the test and let's check that we have metadata for returned columns

Reply:
Added.